.lintrunner.toml

```diff
@@ -116,9 +116,8 @@ include_patterns = [
     '**/*.py',
 ]
 exclude_patterns = [
-    'examples/**',  # TODO: Merge with docs/examples
-    'docs/examples/**',
-    'docs/tutorial/examples/**',
+    'examples/**',
+    'docs/**',
```

Comment:
@justinchuby @gramalingam Are these needed?

Reply:
Yes - otherwise pylint fails - we don't need to run pylint on examples